Crm/PersonAttributeForms.ascx.cs

```diff
@@ -67,6 +67,8 @@ namespace RockWeb.Plugins.rocks_kfs.Crm
     [GroupField( "Group", "Optional group to add person to. If omitted, the group's Guid should be passed via the Query string (GroupGuid=).", false, "", "Groups", 3 )]
     [CustomRadioListField( "Group Member Status", "The group member status to use when adding person to group (default: 'Pending'.)", "2^Pending,1^Active,0^Inactive", true, "2", "Groups", 4 )]
     [BooleanField( "Display SMS Checkbox on Mobile Phone", "Should we show the SMS checkbox when a mobile phone is displayed on the form?", false )]
+    [CustomDropdownListField( "Person Mode", "Person selection mode, should we allow family members, any person guid, or logged in user only.", "Family Members,Anyone,Logged in Person only", true, "Family Members" )]
+    [BooleanField( "Display Family Member Picker", "Should we show the family member picker on the form? (Note: this will only display in Family Members or Anyone mode.) Default: false", false )]
```

Comment:
```suggestion
    [BooleanField( "Display Family Member Picker", "Should we show the family member picker on the form? (Note: this will only display in Family Members or Anyone "Person Mode".) Default: false", false )]
```